cinema/views.py

```diff
@@ -56,3 +95,61 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+
+        if self.action == "list":
+            queryset = (
+                queryset
+                .select_related("movie", "cinema_hall")
+                .annotate(tickets_available=(
+                    F("cinema_hall__rows")
+                    * F("cinema_hall__seats_in_row")
+                    - Count("tickets"))
+                )
+            )
```

Comment:
Reformat this part, currently, there is a mismatch between the presence of arguments for lines with open and closed parenthesis. Take as the rule, that in case there are no arguments on the same line with opening parenthesis, there should be no arguments on the same line with closing as well. In case there are arguments on the same line with the opening parenthesis, there should be arguments on the same line with the closing as well.